### DISCUSSION TITLE
Any way to cut center vocal?

### DISCUSSION THREAD

**Discussion by BaconCondensate** [wwmm/easyeffects#3244] [Q&A]
I want to use my PC as a karaoke machine every so often, but modifying audio files 1 by 1 using an audio editor is not feasible for me.  Are there any way to implement a filter to remove vocal?  Just cutting off the center band using `Stereo Tools` cuts out too much of the music.  I don't need it to be perfect, just decent.  So something like this old audacity filter should in theory work https://plugins.audacityteam.org/nyquist-plugins/effect-plugins/filters-and-eq#details-21 , my question is whether or not there's a way to implement something similar to it in easyeffect?

**Comment by wwmm** [comment]
We have an open issue about this #1400. Unfortunately I still did not have time to study the algorithm used in the Audacity plugin. And just using it as it is is not an option because it isn't a C or C++ library.
